suite-common/icons/package.json

```diff
@@ -25,7 +25,6 @@
         "styled-components": "^6.1.8"
     },
     "devDependencies": {
-        "prettier": "3.2.5",
```

Comment:
Please do not remove prettier from this package, it's used in script for generating icons.

Reply:
Fixed 🫡